lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-cloud_volume.rb

```diff
@@ -1,4 +1,14 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_CloudVolume < MiqAeServiceCloudVolume
+    def backup_create(backup_name, incremental = false, options = {})
+      backup_options = {}
+      backup_options[:name] = backup_name
+      backup_options[:incremental] = incremental
```

Comment:
@tzumainn Can `backup_name` be based as `nil` or an empty string?  Is the `backup_create` method handling that?

Also, you can setup the `backup_options` hash a little easier:

```ruby
backup_options = {
  :name        => backup_name,
  :incremental => incremental
}
```